lib/node_modules/@stdlib/math/base/special/logit/test/test.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/utils/is-nan' );
+var PINF = require( '@stdlib/math/constants/float64-pinf' );
+var NINF = require( '@stdlib/math/constants/float64-ninf' );
+var EPS = require( '@stdlib/math/constants/float64-eps' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var logit = require( './../lib' );
+
+
+// FIXTURES //
+
+var small = require( './fixtures/python/small.json' );
+var medium = require( './fixtures/python/medium.json' );
+var large = require( './fixtures/python/large.json' );
+
+
+// FUNCTIONS //
+
+function almostEqual( a, b, tol ) {
```

Comment:
We have discussed this before. This definition of "almost equal" is wanting. For instance, if `a < 1` and `b < 1`, the tolerance will be too large. Recall that `eps` is the delta between `1` and the __next__ number greater than `1`. As a reminder, this delta is not constant. For smaller values, the delta is less than `eps`.

So, as a reminder, we should __not__ be using this definition. This is particularly the case where, in this definition, `a` and `b` are treated equivalently. Here, this is not true. One value is the reference value and should be the basis for determining the neighborhood of accuracy.

If this definition was used somewhere else, that is a bug.